src/content-distribution/distribute/style.scss

```diff
@@ -0,0 +1,55 @@
+@use "~@wordpress/base-styles/colors";
+
+.newspack-network-distribute {
+	height: calc(100% - 47px);
+	.components-panel {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+	}
+	.components-panel__body {
+		border: 0;
+		&.distribute-header {
+			p {
+				color: colors.$gray-700;
```

Comment:
Sorry to be a pain. Can we just remove this? I tested with the default gray-900 and I actually prefer it.

Reply:
Sure, I prefer it too! e98d9899e8944c9250f6d48ec75602574034ece1